integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -738,6 +738,106 @@ public void testErrorPathDomainWithFailCustomMountCommand() {
     checkPodReady(adminServerPodName, domainUid, domainNamespace);
   }
 
+  /**
+   * Negative Test to create domain with file in auxiliary image not accessible by oracle user.
```

Comment:
Can you describe the usecase with little more detail for easy triaging,

Reply:
added more description